healthcheck/gravitino-healthcheck.sh

```diff
@@ -23,10 +23,8 @@ max_attempts=3
 attempt=0
 success=false
 
-HOST_IP=${GRAVITINO_HOST_IP:-localhost}
-
 while [ $attempt -lt $max_attempts ]; do
-  response=$(curl -X GET -H "Content-Type: application/json" http://${HOST_IP}:8090/api/version)
+  response=$(curl -X GET -H "Content-Type: application/json" http://gravitino:8090/api/version)
```

Comment:
Is it `127.0.0.1` as the hostname inside the gravitino container?

Reply:
yeah,  reverted to `127.0.0.1`.